app/src/main/java/org/simple/clinic/newentry/form/GenderField.kt

```diff
@@ -0,0 +1,10 @@
+package org.simple.clinic.newentry.form
+
+import org.simple.clinic.newentry.form.ValidationError.FieldIsBlankEmpty
+import org.simple.clinic.patient.Gender
+
+class GenderField : InputField<Gender?>() {
```

Comment:
Why is this a `nullable` type?

Reply:
If this is to handle the case where the patient's gender has not yet been selected, shouldn't this be handled at the form level rather than here?

Reply:
Clarification, are you talking about the impending `Form` object or the UI layer? If we are discussing the former, it will show up in an upcoming PR. If we are discussing the later, then we longer have a single-source of truth.

Reply:
I meant the `Form` object.

Reply:
I am assuming that the `GenderField` will become an `InputField<Gender>`, and the impending `PatientEntryForm` object should potentially have either an `GenderField?` or an `Optional<GenderField>` property.

Does this make sense?